nevergrad/benchmark/optgroups.py

```diff
@@ -46,6 +47,22 @@ def large() -> tp.Sequence[Optim]:
             "MultiCMA", "TripleCMA", "MultiScaleCMA", "RSQP", "RCobyla", "RPowell", "SQPCMA", "MetaModel", "PolyCMA", "ManyCMA"]
 
 
+@registry.register
+def emna_variants() -> tp.Sequence[Optim]:
+    return ["IsoEMNA", "NaiveIsoEMNA", "AnisoEMNA", "NaiveAnisoEMNA", "CMA", "NaiveTBPSA",
+              "NaiveIsoEMNATBPSA", "IsoEMNATBPSA", "NaiveAnisoEMNATBPSA", "AnisoEMNATBPSA"]
+
+
+@registry.register
+def progressive() -> tp.Sequence[Optim]:
+    return ["ProgONOPOInf", "ProgONOPOAuto", "ProgONOPO13", "ProgODOPOInf", "ProgODOPOAuto", "ProgODOPO13"]
```

Comment:
Here is what I would prefer, creating all the combinations with for loops in optgroups:

```python
optims:tp.List[Optim] = []
for mutation in ["discrete", "gaussian"]:
    for num_optims in [None, 13, 10000]:
        name = "Prog" + ("Disc" if mutation == "discrete" else "") + ("Auto" if num_optims is None else str(num_optims)
        mv = ParametrizedOnePlusOne(noise_handling="optimistic", mutation=mutation)
        opt = ConfSplitOptimizer(
            num_optims=num_optims, progressive=True, multivariate_optimizer=mv
        ).set_name(name))
        optims.append(opt)
return optims
```
 Why?
- because it removes a whole lot of optim definition with unreadable names
- because it is easier to extend/adapt
- because it is less likely to be buggy

Reply:
Done.